README.md

```diff
@@ -94,3 +94,37 @@ console.error();
 console's `log`, `info`,`dubug` reports as info log.
 
 console's `error`, `warn` reports as error log if message contains _error_ mention, otherwise as warn log.
+
+### Reporting API
+
+This reporter provides Reporting API to use it directly in tests to send some additional data to the report.
+
+To start using the `ReportingApi` in tests, just import it from `'@reportportal/agent-js-vitest'`:
+```javascript
+import { ReportingApi } from '@reportportal/agent-js-vitest';
+```
+
+#### Reporting API methods
+
+The API provide methods for attaching data.<br/>
+
+##### attachment
+Send file to report portal for the current test. Should be called inside of corresponding test.<br/>
```

Comment:
We prefer using the same product naming across all its component, so let's change `report portal` to `ReportPortal` to be consistent.
```suggestion
Send file to ReportPortal for the current test. Should be called inside of corresponding test.<br/>
```